Assets/Text/cqui_text_settings.xml

```diff
@@ -178,10 +178,10 @@
       <Text>Toggles the popup that appears whenever a new tech or civic is achieved</Text>
     </Row>
     <Row Tag="LOC_CQUI_POPUPS_TECHAUDIO" Language="en_US">
-      <Text>Tech/Civic popup audio</Text>
+      <Text>Auto-play Tech/Civic popup audio</Text>
     </Row>
     <Row Tag="LOC_CQUI_POPUPS_TECHAUDIO_TOOLTIP" Language="en_US">
-      <Text>Toggles the popup audio that plays whenever a new tech or civic is achieved. Is fully independent of the visual component and can play even when there is no visible popup.</Text>
+      <Text>Toggles automatically playing the audio voiceover when the Tech/Civic popup appears after completing a tech or civic</Text>
```

Comment:
Since this text is dofferent than the original one then all translated texts also are wrong :(